ecosystem/sep-0040.md

```diff
@@ -0,0 +1,160 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Non-Fungible Token Standard
+Author: George Kudrayvtsev <george@stellar.org>
+Track: Standard
+Status: Draft
+Created: 2021-01-31
+Updated: 2021-01-31
+Version: 0.1.0
+Discussion: TBD
+```
+
+## Simple Summary
+This SEP defines a standard interface to describe non-fungible tokens (NFTs) on the Stellar network, including a distribution mechanism and metadata fields. It draws inspiration from the [EIP-721 standard](https://eips.ethereum.org/EIPS/eip-721).
+
+
+## Motivation
+A standard interface for NFTs allows ecosystem applications (such as wallets or marketplaces) to create, view, or otherwise work with any NFT on Stellar.
+
+
+## Abstract
+Non-fungible tokens (NFTs) can describe many types of assets, from digital collectibles to physical items. We should also define a standard issuance model to explain how NFTs can and should be issued on the network. In order to maintain compatibility across the Stellar ecosystem and beyond, we need a standard way to describe and define them that is agnostic of what the NFT actually describes.
```

Comment:
The `We should define` is somewhat confusing, should we define it, or is it defined by the protocol? It looks like the protocol defines it, so I think we can say it defines it. Suggested rewording:
```suggestion
Non-fungible tokens (NFTs) can describe many types of assets, from digital collectibles to physical items. This protocol defines a standard set of steps that NFT issuers should use to issue NFTs on the Stellar network, defines metadata that describes an NFT, and specifies how the metadata is linked to the NFT asset on Stellar.
```

Reply:
I think the `should` there was an editing mishap. I do like your reword, though: it's a good thesis sentence.